core/src/main/scala/org/ensime/core/Project.scala

```diff
@@ -47,19 +47,35 @@ class Project(
   private implicit val vfs: EnsimeVFS = EnsimeVFS()
   private val resolver = new SourceResolver(config)
   private val searchService = new SearchService(config, resolver)
-  private val sourceWatcher = new SourceWatcher(config, resolver :: Nil)
+
+  // map to get all projects that list the key project in 'depends'
+  private val dependentProjects: Map[EnsimeProjectId, List[EnsimeProjectId]] =
```

Comment:
sadly, yes (I know these rules about `var` are inconsistent, but that's `Actor`s for you...)